honeybee_energy/lib/_loadconstructions.py

```diff
@@ -0,0 +1,35 @@
+"""Load all materials and constructions from the IDF libraries."""
+from honeybee_energy.construction import OpaqueConstruction, \
+    WindowConstruction
+
+import os
+import inspect
+
+
+# empty dictionaries to hold idf-loaded materials and constructions
+_idf_opaque_materials = {}
+_idf_window_materials = {}
+_idf_opaque_constructions = {}
+_idf_window_constructions = {}
+
+
+# load other materials and constructions from user-supplied files
+cur_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+construction_lib = cur_dir + '/idf/constructions/'
```

Comment:
I would use os.path.join in such cases. It should work on Windows like this but os.path.join is cleaner.

```suggestion
construction_lib = os.path.join(cur_dir, 'idf', 'constructions')
```